core/server/common/src/main/java/alluxio/cli/Format.java

```diff
@@ -48,13 +50,20 @@
     WORKER,
   }
 
-  private static void formatFolder(String name, String folder) throws IOException {
+  private static void formatWorkerDataFolder(String name, String folder) throws IOException {
     LOG.info("Formatting {}:{}", name, folder);
```

Comment:
It's odd to pass a string to the method only for the purpose of including that string in the log message. It requires some coordination between the caller and the method so that the error message sounds good. I think it would be better to remove the `name` argument here, and do the logging at the call site.